### PR TITLE
feat: DAH-2120 add fixtures

### PR DESCRIPTION
**Pull request by tallulahkay**
DAH-2120

### CODE REVIEW

**Comment by jimlin-sfgov** [comment]
suggestion: add the cypress path to `exclude_patterns` in `.codeclimate.yml`

File: cypress/e2e/SupplementalApplicationPage/status_update.e2e.js [hunk 1]
```diff
@@ -1,25 +1,67 @@
...
     cy.visit('http://localhost:3000/')
     cy.login()
     cy.visit(`/lease-ups/applications/${LEASE_UP_LISTING_APPLICATION_ID}`)
+    cy.wait('@shortForm')
+    cy.wait('@fieldUpdateCommentsGet')
+    cy.wait('@leases')
+    cy.wait('@rentalAssistances')
+    cy.wait('@supplementals')
+    cy.wait('@units')
+    cy.wait('@leaseUpListing')
```
Comment (jimlin-sfgov) [inline cypress/e2e/SupplementalApplicationPage/status_update.e2e.js:72]: issue: this test makes a POST request to the `field_update_comments` endpoint, that is not getting intercepted. We need to create an additional intercept for this one.

**Review by jimlin-sfgov — CHANGES_REQUESTED** [review]

File: cypress/e2e/SupplementalApplicationPage/status_update.e2e.js (hunk 1 above)
Comment (tallulahkay) [reply cypress/e2e/SupplementalApplicationPage/status_update.e2e.js:72]: Thank you for catching this! Looking into it right now

Comment (tallulahkay) [reply cypress/e2e/SupplementalApplicationPage/status_update.e2e.js:72]: I added the missing intercept!

**Review by jimlin-sfgov — APPROVED** [review]
nicely done!

**Pull request merged**